lib/user.rb

```diff
@@ -0,0 +1,16 @@
+class User
+  attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(jokes)
+    @jokes << jokes
+  end
+
+  def tell(name, jokes)
+    name.learn(jokes)
```

Comment:
good start here. Two minor things: we're only passing a single joke here, and typically anything plural that you pass as a parameter can make people think you're taking in an array of those things instead of a single object. Second, we should only be able to tell someone a joke that we've learned ourself.